test/tests.py

```diff
@@ -533,7 +503,6 @@ def test_flags(self, filename=None, append_to=None):
             key = key.decode('utf-8')
             if data:
                 data = data.decode('utf-8')
-            self.pr("%s : %s" % (key, data))
```

Comment:
This print is harmless and _sometimes somewhat_ useful to glance-debug why a test you're writing isn't working. I vote to keep it.

Reply:
The problem is that we call this code earlier than before, when self.pr does not yet work.

Reply:
Now replaced with this line, printed somewhat later:
```
flags: DMLFileTestCase.TestFlags(exp_warnings=[], exp_errors=[('minimal.dml', 6, 'EIMPORT')], exp_stdout=[], cc_flags=[], dmlc_flags=[], api_version='6', instantiate_manually=False, compile_only=False, no_cc=False)
```